Roys/app/src/main/java/com/dwg_karrier/roys/ScriptedURL.java

```diff
@@ -0,0 +1,26 @@
+package com.dwg_karrier.roys;
+
+public class ScriptedURL {
+  String url;
+  boolean isRead;
+
+  ScriptedURL(String url, boolean isRead) {
+    this.url = url;
+    this.isRead = isRead;
+  }
+
+  ScriptedURL(String url, int isRead) {
+    this.url = url;
+
+    if (isRead==1) {
```

Comment:
https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace
space between variable / constant and operator